backend/assignment-service/src/routes/questions/index.ts

```diff
@@ -0,0 +1,40 @@
+import express from "express";
+import { QuestionController } from "../../controllers/question-controller";
+
+const router = express.Router();
+
+router
+  .route("/assignments/:id/questions")
+  .post(QuestionController.createQuestion);
+
+router.route("/questions/:questionId").get(QuestionController.getQuestionById);
```

Comment:
Same as the previous comment

Reply:
I think the issue here is that since putting this code in one line is too long, so the auto-formatter arrange the code this way :)